### PR TITLE
Fix names

### PR DESCRIPTION
**Pull request by charisprose**

### CODE REVIEW

**Review by becky-intelletive — CHANGES_REQUESTED** [review]
- Kubesec should be capitalized [Kubesec repo](https://github.com/controlplaneio/kubesec)
- kube-bench should be lowercase with a dash [kube-bench repo](https://github.com/aquasecurity/kube-bench)
- Gatekeeper should be capitalized [Gatekeeper repo](https://github.com/open-policy-agent/gatekeeper)
- kube-hunter should be lowercase with a dash [kube-hunter repo](https://github.com/aquasecurity/kube-hunter)

File: dash/backend/src/gatekeeper-templates/Readme.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Gatekeeper Libraries
+# gatekeeper Libraries
```
Comment (jacobbeasley) [inline dash/backend/src/gatekeeper-templates/Readme.md:1]: https://github.com/open-policy-agent/gatekeeper

Looks like Gatekeeper can be capitalized (based upon what I'm seeing in their github)

File: docs/content/en/docs/Getting started/advanced-install.md [hunk 2]
```diff
@@ -80,79 +80,79 @@ rabbitmq:
 The following table lists the configurable parameters of the chart and the default values.
 
 
-| Parameter                                                                   | Description                                                                                                        | Default                         |
```
Comment (becky-intelletive) [inline docs/content/en/docs/Getting started/advanced-install.md:83]: Was changing this intentional?

**Review by becky-intelletive — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/Getting started/advanced-install.md (hunk 2 above)
Comment (charisprose) [reply docs/content/en/docs/Getting started/advanced-install.md:83]: The current table was not format correctly.
Before:
Screenshot 2023-08-02 at 8 55 13 AM
Screenshot 2023-08-02 at 8 54 10 AM

**Review by becky-intelletive — APPROVED** [review]

**Pull request merged**